components/lvgl_esp32_drivers/lvgl_tft/ili9341.c

```diff
@@ -159,6 +159,12 @@ void ili9341_init(void)
 	ili9341_send_cmd(0x36);
 	ili9341_send_data(&data, 1);
 #endif
+
+#if defined ILI9341_INVERT_COLORS
```

Comment:
Check `build/config/sdkconfig.h` to know what value `CONFIG_LVGL_INVERT_COLORS` have if you set to option to false, I don't know if the option is set to 0 or simply not defined.

If it is set to 0 when you don't enable the option then I think this must be:
```c
#if ILI9341_INVERT_COLORS == 1
```

Reply:
The `CONFIG_LVGL_INVERT_COLORS` value equals `1` when it's set and it's undefined when not set.

I changed it to check the value and it's working correctly both when the color inversion option is set and when it is not set.